tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -31,6 +31,22 @@
 # todo: make sure '_' works in parameter names in all cases, add a warning if not
 
 
+def _roll(a, shift):
```

Comment:
can you please add a few unit tests for this method?

Reply:
Also, you designed this function with a certain goal in mind (I want to replace `np.roll` as its not fast enough). This reason needs to go into the docstring! This will help other people greatly.

Finally, it would be nice to know what the shift parameter does, like what happens if I use -1 or 1, in which direction is the method shifting the input.

Reply:
for the unit tests you could just extract some random arrays and show that it is equal to numpys.roll